core/templates/dev/head/pages/exploration_editor/save_exploration_modal.html

```diff
@@ -0,0 +1,31 @@
+<script type="text/ng-template" id="modals/saveExploration">
```

Comment:
Has the existing code for this modal been deleted? Alternatively, you could get rid of this file and deal with it in a separate PR if it's related to @Keksike's changes.